Modules/Bridge/NumPy/wrapping/PyBuffer.i.init

```diff
@@ -1,7 +1,7 @@
 %pythoncode %{
 
 import numpy as np
-class NDArrayITKBase(np.ndarray):
+class NDArrayITKBase(np.ndarray) -> np.dtype:
```

Comment:
Remove per:

```
Traceback (most recent call last):
  File "/Users/runner/work/1/s/Modules/Core/Common/wrapping/test/itkMatrixTest.py", line 28, in <module>
    image.SetDirection(new_direction)
  File "/Users/runner/work/1/s-build/Wrapping/Generators/Python/itk/itkImagePython.py", line 4397, in SetDirection
    matrix = itk.matrix_from_array(array)
  File "/Users/runner/work/1/s-build/Wrapping/Generators/Python/itk/support/extras.py", line 612, in GetMatrixFromArray
    vnl_matrix = GetVnlMatrixFromArray(arr)
  File "/Users/runner/work/1/s-build/Wrapping/Generators/Python/itk/support/extras.py", line 592, in GetVnlMatrixFromArray
    return _GetVnlObjectFromArray(arr, "GetVnlMatrixFromArray", ttype)
  File "/Users/runner/work/1/s-build/Wrapping/Generators/Python/itk/support/extras.py", line 575, in _GetVnlObjectFromArray
    keys = [k for k in itk.PyVnl.keys() if k[0] == PixelType]
  File "/Users/runner/work/1/s-build/Wrapping/Generators/Python/itk/support/lazy.py", line 138, in __getattribute__
    base.itk_load_swig_module(module, namespace)
  File "/Users/runner/work/1/s-build/Wrapping/Generators/Python/itk/support/base.py", line 132, in itk_load_swig_module
    l_module = loader.load(swig_module_name)
  File "/Users/runner/work/1/s-build/Wrapping/Generators/Python/itk/support/base.py", line 291, in load
    l_spec.loader.exec_module(l_module)  # pytype: disable=attribute-error
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/Users/runner/work/1/s-build/Wrapping/Generators/Python/itk/support/../ITKBridgeNumPyPython.py", line 71, in <module>
    from itk.itkPyVectorContainerPython import *
  File "/Users/runner/work/1/s-build/Wrapping/Generators/Python/itk/itkPyVectorContainerPython.py", line 103
    class NDArrayITKBase(np.ndarray) -> np.dtype:
```

Reply:
Add return type to: `_get_numpy_pixelid`

Reply:
Oversight on my part. Updated.